default/stringtables/en.txt

```diff
@@ -1384,6 +1384,12 @@ Set the minimum number of human players required to start a multiplayer game.
 OPTIONS_DB_MP_HUMAN_MAX
 Limit the number of human players allowed in a multiplayer game. Set to -1 for unlimited.
 
+OPTIONS_DB_MP_CONN_HUMAN_MIN
+Server stop the playing game if count of active human player will be less than this value.
```

Comment:
Minimum number of active human players, below which the server will stop the game being played.

Reply:
Fixed.